packages/typo3-guides-cli/src/Command/InitCommand.php

```diff
@@ -0,0 +1,247 @@
+            $workdir = $input->getOption('working-dir');
+            assert(is_string($workdir));
+
+            if (chdir($workdir)) {
+                $output->writeln('<info>Changed working directory to ' . getcwd() . '</info>');
+            } else {
+                $output->writeln('<error>Could not change working directory to ' . $workdir . '</error>');
+                return Command::INVALID;
+            }
+        }
+
+        if ($input->getOption('quiet')) {
+            echo 'This command is interactive and requires user input.' . PHP_EOL;
+            return Command::INVALID;
+        }
+
+        if (file_exists('Documentation/guides.xml')) {
+            $output->writeln('<error>A "Documentation" directory already exists in this directory</error>');
+            return Command::INVALID;
+        }
+
+        $output->writeln('Welcome to the <comment>TYPO3 documentation</comment> project setup wizard');
+        $output->writeln('This wizard will help you to create a new documentation project in the current directory.');
+        $output->writeln('');
+
+        $composerInfo = $this->getComposerInfo($output);
+
+        /** @var QuestionHelper $helper */
+        $helper = $this->getHelper('question');
+
+
+        $question = new Question(sprintf('Do you want to use reStructuredText(rst) or MarkDown(md)? <comment>[rst, md]</comment>: '), 'rst');
+        $question->setValidator(function ($answer) {
+            if (is_null($answer) || !in_array($answer, ['rst', 'md'], true)) {
+                throw new \RuntimeException('Choose reStructuredText(rst) or MarkDown(md). ');
+            }
+            return $answer;
+        });
+        $format = $helper->ask($input, $output, $question);
+
+        $projectNameQuestion = new Question(sprintf('What is the title of your documentation? <comment>[%s]</comment>: ', $composerInfo?->getComposerName()), $composerInfo?->getComposerName());
+        $projectNameQuestion->setValidator(function ($answer) {
+            if (is_null($answer) || trim($answer) === '') {
+                throw new \RuntimeException('The project title cannot be empty.');
+            }
+            return $answer;
+        });
+
+        $projectName = $helper->ask($input, $output, $projectNameQuestion);
+
+        $question = $this->createValidatedUrlQuestion(
+            sprintf('What is the URL of your project\'s homepage? <comment>[%s]</comment>: ', $composerInfo?->getHomepage()),
+            $composerInfo?->getHomepage(),
+            ['https://extensions.typo3.org/package/' . $composerInfo?->getComposerName()]
+        );
+        $projectHomePage = $helper->ask($input, $output, $question);
+
+
+        $question = $this->createValidatedUrlQuestion(
+            sprintf(sprintf('What is the URL of your project\'s repository?   <comment>[%s]</comment>', 'https://github.com/' . $composerInfo?->getComposerName()), 'https://github.com/' . $composerInfo?->getComposerName()),
```

Comment:
Can you wrap into new lines here? We have a max. line length of 120.

Reply:
unless there is a way to do this automatically I would leave it like that...